testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/authentication/ProvidersTest.java

```diff
@@ -141,7 +142,7 @@ private List<Map<String, Object>> expectedAuthProviders() {
                 "Validates a OTP on a separate OTP form. Only shown if required based on the configured conditions.");
         addProviderInfo(result, "auth-cookie", "Cookie", "Validates the SSO cookie set by the auth server.");
         addProviderInfo(result, "auth-otp-form", "OTP Form", "Validates a OTP on a separate OTP form.");
-        if (Profile.isFeatureEnabled(Profile.Feature.SCRIPTS)) {
+        if (ProfileAssume.isFeatureEnabled(Profile.Feature.SCRIPTS)) {
```

Comment:
Has to use ProfileAssume, not Profile, as it needs to be checked server-side.

With the changes in the Profile init this will also be caught as Profile won't be initialized resulting in a NPE, so this mistake won't happen again in the future.